dependencies.yaml

```diff
@@ -43,3 +43,12 @@ libocpp:
 Josev:
   git: https://github.com/EVerest/ext-switchev-iso15118.git
   git_tag: 2023.1.0
+
+# OpenV2G
+ext-openv2g:
+  git: https://github.com/EVerest/ext-openv2g.git
+  git_tag: everest
+# mbedtls
+libmbedtls:
+  git: https://github.com/chargebyte/mbedtls.git
```

Comment:
Please change to:
`git: https://github.com/EVerest/ext-mbedtls.git`

This is a fork from your mbedtls repo.